mod/InGameTracker/Logic.cs

```diff
@@ -354,6 +357,31 @@ public void DetermineAllAccessibility()
                 break;
             }
 
+            // Check for Brittle Hollow warp connections
+            switch (w1)
+            {
+                case "BHF":
+                    bhfConnection = w2;
+                    break;
+                case "BHNG":
+                    bhngConnection = w2;
+                    break;
+                case "WHS":
+                    whsConnection = w2;
+                    break;
+            }
+            switch(w2) {
+                case "BHF":
+                    bhfConnection = w1;
+                    break;
+                case "BHNG":
+                    bhngConnection = w1;
+                    break;
+                case "WHS":
+                    whsConnection = w1;
+                    break;
+            }
```

Comment:
I know this isn't the official standard, but when cases are _this_ trivial I strongly prefer to make them one-liners / not waste so many extra lines
```suggestion
            switch (w1)
            {
                case "BHF": bhfConnection = w2; break;
                case "BHNG": bhngConnection = w2; break;
                case "WHS": whsConnection = w2; break;
            }
            switch(w2) {
                case "BHF": bhfConnection = w1; break;
                case "BHNG": bhngConnection = w1; break;
                case "WHS": whsConnection = w1; break;
            }
```